### ISSUE TITLE
Fix duplicated drug indications

### ISSUE THREAD

**Issue by mkarmona** [opentargets/platform#929]
The drug `CHEMBL657` contains the indication of `pain` more than once. So finding a clear answer to this behaviour is the aim.

**Comment by d0choa** [comment]
When looking at the ChEMBL indications for the same drug [here](https://www.ebi.ac.uk/chembl/g/#browse/drug_indications/filter/drug_indication.parent_molecule_chembl_id%3ACHEMBL657), they list 47 indications, whereas we list 49.

The reason it's because we use the `parent_molecule_chembl_id` so in a way it's a grouping of all the indications for several molecules.

```
❯ jq -r 'select((.parent_molecule_chembl_id == "CHEMBL657") and (.mesh_heading == "Pain"))' chembl_drug_indication_rest_api-2020-06-01.json
{
  "efo_id": "EFO:0003843",
  "drugind_id": 62990,
  "efo_term": "pain",
  "indication_refs": [
    {
      "ref_type": "ClinicalTrials",
      "ref_url": "https://clinicaltrials.gov/search?id=%22NCT03396250%22",
      "ref_id": "NCT03396250"
    }
  ],
  "parent_molecule_chembl_id": "CHEMBL657",
  "molecule_chembl_id": "CHEMBL1620",
  "mesh_id": "D010146",
  "mesh_heading": "Pain",
  "max_phase_for_ind": 1
}
{
  "efo_id": "EFO:0003843",
  "drugind_id": 68356,
  "efo_term": "pain",
  "indication_refs": [
    {
      "ref_type": "ClinicalTrials",
      "ref_url": "https://clinicaltrials.gov/search?id=%22NCT02229539%22",
      "ref_id": "NCT02229539"
    }
  ],
  "parent_molecule_chembl_id": "CHEMBL657",
  "molecule_chembl_id": "CHEMBL657",
  "mesh_id": "D010146",
  "mesh_heading": "Pain",
  "max_phase_for_ind": 3
}
```

This is not the intended behaviour. The actions to fix this issue are the next:

1. Make sure that the indications are unique. Group by `efo_id` and `parent_molecule_chembl_id` and take max phase of the resulting grouping. `Pain` in `CHEMBL657` should only appear once with a `max_phase_for_ind` of 3. 

2. Similarly, concatenate all the `indication_refs` and include them in the index / endpoint. This information is currently not available.

**Comment by d0choa** [comment]
@andrewhercules `indication_refs` will be available in ES when pipeline is run for 20.09. With minor API and FE work we could add the references to the indications (similar to ChEMBL).
